test/es-module/test-esm-experimental-warnings.mjs

```diff
@@ -25,7 +25,10 @@ describe('ESM: warn for obsolete hooks provided', { concurrency: true }, () => {
   describe('experimental warnings for enabled experimental feature', () => {
     for (
       const [experiment, arg] of [
-        [/Custom ESM Loaders/, `--experimental-loader=${fileURL('es-module-loaders', 'hooks-custom.mjs')}`],
+        [
+          /`--experimental-loader` may be removed in the future/,
+          `--experimental-loader=${fileURL('es-module-loaders', 'hooks-custom.mjs')}`
```

Comment:
Let’s use two separate arguments in case the file URL contains "weird" chars
```suggestion
          '--experimental-loader',
          fileURL('es-module-loaders', 'hooks-custom.mjs'),
```